### ISSUE TITLE
ImportError: No module named site, Error in normalizePath(path.expand(path), winslash, mustWork) 

### ISSUE THREAD

**Issue by achoknyz** [ranghetti/sen2r#323]
I have made a clean installation of the software and properly made the paths to directories but I don't seem to save any outputs (indices & rgb) as set. Ave attached the progress print from the rstudio.
[Sen2r error.pdf](https://github.com/ranghetti/sen2r/files/4728517/Sen2r.error.pdf)

**Comment by ranghetti** [comment]
This error is due to an oversight of mine (I fixed it on Windows, forgetting to push the edits). Please try reinstalling the package with
```r
install.packages("remotes")
remotes::install_github("ranghetti/sen2r", ref = "devel")
```
As soon as you will confirm me that this will fix your error, I will update the CRAN version.

**Comment by ranghetti** [comment]
A new version was released on CRAN; you can install and test it using
```r
install.packages("sen2r", type = "source")
```

Reopen this issue if it will not solve (or open a new one in case of a different unrelated error).

**Comment by achoknyz** [comment]
Still have errors. A new error related to GDAL. Still no saving of the different outputs specified
[new Sen2r error.pdf](https://github.com/ranghetti/sen2r/files/4729838/new.Sen2r.error.pdf)

**Comment by ranghetti** [comment]
This is a different unrelated error, please open a new issue following the indications in #186.

**Comment by achoknyz** [comment]
I have made a fresh installation from "install.packages("sen2r", type = "source")" the part of the challenge solved. What still remains is that the process does not save my desired out indices. It manages to save level 2, output (BOA, SCL) and the RGB but not the indices. I have attached the required information
![Reportoutput](https://user-images.githubusercontent.com/5365067/83800719-d6ae9d00-a69f-11ea-8ead-e2cd020a3098.JPG)
![SessionInfo](https://user-images.githubusercontent.com/5365067/83800721-d7473380-a69f-11ea-9915-59e200bcf76b.JPG)
[Test2_JSONfile.txt](https://github.com/ranghetti/sen2r/files/4732252/Test2_JSONfile.txt)
